website/docs/reference/sinks/gcp_cloud_storage.md.erb

```diff
@@ -0,0 +1,21 @@
+<%- component = metadata.sinks.gcp_cloud_storage -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+## Options
+
+<%= fields(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list) %>
+
+<%- end -%>
+## How It Works [[sort]]
```

Comment:
I'd like to add similar sections to what we have in the `aws_s3` sink. Specifically:

1. Cross account object writing
2. Object access control list (ACL) -- or whatever GCP's terminology is
3. Object naming
4. Server-side encryption (SSE) - if relevant
5. Storage class

Let me know if you disagree or it feels overkill. I don't understand this sink as well as S3.

Reply:
1. I don't think cross account object writing is possible. I did not find a reference to support for it in the XML API documentation.
2. We support predefined ACLs as per the link in the `.meta` file.
3. Object naming is the same as the `aws_s3` sink, so yes that should get added here.
4. I dropped support for server-side encryption for this attempt. It works differently than `aws_s3` with different options.
5. Storage class definitely needs adding.

Reply:
Sections added.